package.json

```
@@ -25,22 +25,22 @@
     "react-helmet-async": "^1.2.2",
     "react-i18next": "^11.15.2",
     "react-infinite-scroll-component": "^6.1.0",
-    "react-leaflet": "^3.2.2",
+    "react-leaflet": "^3.2.4",
     "react-leaflet-markercluster": "^3.0.0-rc1",
-    "react-papaparse": "^3.17.2",
+    "react-papaparse": "^3.18.2",
     "react-router": "^6.2.1",
     "react-router-dom": "^6.2.1",
-    "react-scripts": "4.0.3",
+    "react-scripts": "^5.0.0",
     "react-style-proptype": "^3.2.2",
     "slugify": "^1.6.4",
     "uuid": "^8.3.2",
     "web-vitals": "^2.1.2"
   },
   "scripts": {
-    "start": "cross-env SKIP_PREFLIGHT_CHECK=true DISABLE_ESLINT_PLUGIN=true react-scripts start",
-    "build": "cross-env SKIP_PREFLIGHT_CHECK=true DISABLE_ESLINT_PLUGIN=true react-scripts build",
+    "start": "cross-env react-scripts start",
+    "build": "cross-env react-scripts build",
```

~~cross-env is not really needed now anymore so we can simply remove it from all the commands. I would also suggest to remove the module all together if it's not used elsewhere.~~

This needs to be added back in order for build to work:

```suggestion
    "start": "cross-env DISABLE_ESLINT_PLUGIN=true react-scripts start",
    "build": "cross-env DISABLE_ESLINT_PLUGIN=true react-scripts build",
```

---

OR at least on the build command. It works locally for me without disabling the eslint plugin but in the docker container it fails. I can't say why.

---

OK, I have found the issue. This is the line that is causing the build to fail:

https://github.com/sledilnik/zdravniki/blob/master/.dockerignore#L5

Basically `.prettierrc.json` is not copied to the docker image and therefore prettier inside of the docker fails as it has a different configuration. I would suggest removing it.

Afterwards, it is also safe to remove all the `cross-env` it seems.

---

@jalezi let me know if you need anymore help.